app/src/main/java/com/eveningoutpost/dexdrip/LibreAlarmReceiver.java

```diff
@@ -55,6 +57,7 @@
     private static final Object lock = new Object();
     private static long sensorAge = 0;
     private static long timeShiftNearest = -1;
+    private static final long days_in_msec =  24 * 60* 60 * 1000;
```

Comment:
Its only a small thing, but there is `Constants.DAY_IN_MS`

Reply:
Done.